src/targets/stm32f1-generic/main.c

```diff
@@ -5,9 +5,13 @@
 
 #include <stm32f1xx.h>
 
+#include "platform/stm32f1/flash.h"
 #include "platform/stm32f1/rcc.h"
 
 int main()
 {
+	flash_latency_config(72000000);
+
 	rcc_init(EXTERNAL_CLOCK_VALUE);
+	rcc_update_clock_tree();
```

Comment:
Hum, actually, maybe this should be called by `rcc_init`?

Reply:
`rcc_init_clock_tree`? if so i prefer `update`

Reply:
No, I mean that when we init the clocks we should update the tree afterwards, instead of having to do it later outside the function. I think we should call `rcc_update_clock_tree` at the end of `rcc_init` :laughing:

Reply:
ah, sure, why not